### DISCUSSION TITLE
OctoText is shown as "incompatible"

### DISCUSSION THREAD

**Discussion by Master-68** [berrystephenw/OctoText#27] [Show and tell]
...though current OctoPrint Versio 1.5.3

browser.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:87.0) Gecko/20100101 Firefox/87.0
connectivity.connection_check: 8.8.8.8:53
connectivity.connection_ok: true
connectivity.enabled: true
connectivity.online: true
connectivity.resolution_check: octoprint.org
connectivity.resolution_ok: true
env.hardware.cores: 4
env.hardware.freq: 1500
env.hardware.ram: 3959984128
env.os.bits: 32
env.os.id: linux
env.os.platform: linux2
env.plugins.pi_support.model: Raspberry Pi 4 Model B Rev 1.2
env.plugins.pi_support.octopi_version: 0.17.0
env.plugins.pi_support.throttle_state: 0x0
env.python.pip: 19.3.1
env.python.version: 2.7.16
env.python.virtualenv: true
octoprint.safe_mode: false
octoprint.version: 1.5.3

![OctoText](https://user-images.githubusercontent.com/71970028/113287308-b36b7300-92ed-11eb-91a4-273e7dc68200.png)

**Comment by berrystephenw** [comment]
Hi - sorry you are having trouble. The problem is that your Octoprint distribution is running Python 2.7:
` env.python.version: 2.7.16`
OctoText relies on a Python 3 distribution and is not compatible with the older Python 2. 
If you still want to use OctoText (and your other plugins are compatible with Python 3) then you can follow this blog post to upgrade:
https://community.octoprint.org/t/upgrade-your-octoprint-install-to-python-3/23973

I hope this helps!
